pkg/tsdb/sql_engine.go

```diff
@@ -98,8 +98,11 @@ var NewSqlQueryEndpoint = func(config *SqlQueryEndpointConfiguration, rowTransfo
 		return nil, err
 	}
 
-	engine.SetMaxOpenConns(10)
-	engine.SetMaxIdleConns(10)
+	maxConns := config.Datasource.JsonData.Get("maxConnections").MustInt(0)
+	if maxConns > 0 {
+		engine.SetMaxOpenConns(maxConns)
+		engine.SetMaxIdleConns(maxConns)
+	}
```

Comment:
this appears to change the logic here quite a bit, removing the SetMaxOpenConns and SetMaxIdelConns calls

Reply:
Agree, it was 10 earlier for both SetMaxOpenConns and SetMaxIdleConns.